kernel/posix/pthread.c

```diff
@@ -0,0 +1,537 @@
+/*
+ * Copyright (c) 2018 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <kernel.h>
+#include <pthread.h>
+#include <stdio.h>
+#include <atomic.h>
+#include "ksched.h"
+#include "wait_q.h"
+
+const pthread_attr_t init_pthread_attrs = {
```

Comment:
This should be static. It's an implementation detail that's used only by functions in this .c file

Reply:
Yes it should be static.